debezium-core/src/main/java/io/debezium/transforms/ByLogicalTableRouter.java

```diff
@@ -178,6 +180,8 @@ public void configure(Map<String, ?> props) {
         }
         keyFieldName = config.getString(KEY_FIELD_NAME);
         keyEnforceUniqueness = config.getBoolean(KEY_ENFORCE_UNIQUENESS);
+
+        smtManager = new SmtManager<>(config);
```

Comment:
do we need a setter for tests?

Reply:
I don't think so. The config can be passed in, and I don't see a need for using an alternative impl of that utility class.